### PR TITLE
IBN-2262 changes default hue polling interval from 10s to 1s

### PR DESCRIPTION
**Pull request by pwolfart**

### CODE REVIEW

**Comment by pwolfart** [comment]
This fixes our problems but I fear this is a brute force fix. 
Does anyone know a different way to set this? Shouldn't there be a configuration file for the binding?

**Comment by mkhl** [comment]
> Does anyone know a different way to set this?

Since this is configurable per bridge, I think the intended way to modify the polling interval is to set it in the bridge’s configuration.

> Shouldn't there be a configuration file for the binding?

It might make sense, but since these are default values that can already be changed per bridge, it might not have seemed necessary.

File: extensions/binding/org.eclipse.smarthome.binding.hue/ESH-INF/thing/bridge.xml [hunk 1]
```diff
@@ -32,7 +32,7 @@
 				<label>Polling Interval</label>
 				<description>Seconds between fetching values from the Bridge.</description>
 				<required>true</required>
-				<default>10</default>
+				<default>1</default>
```
Comment (mkhl) [inline extensions/binding/org.eclipse.smarthome.binding.hue/ESH-INF/thing/bridge.xml:35]: Note that there are two more occurrences of the ten second default value, one in HueBridgeConfig and one in HueBridgeHandler. When going with this approach, we might want to change them as well to stay consistent.

**Review by mkhl — COMMENTED** [review]
The title and commit message should indicate that this changes the **default** polling interval.

File: extensions/binding/org.eclipse.smarthome.binding.hue/ESH-INF/thing/bridge.xml (hunk 1 above)
Comment (pwolfart) [reply extensions/binding/org.eclipse.smarthome.binding.hue/ESH-INF/thing/bridge.xml:35]: That is not very consistent. However, I think this is the route to go. 
Once upon a time we set this value at runtime and suffered from the fact that it was not applied instantaneously. Instead a restart of the framework was required to be applied. 

I'm going to change values in HueBridgeConfig and HueBridgeHandler, too.

**Review by mkhl — COMMENTED** [review]
I still think that the title and commit message should indicate that this changes the **default** polling interval.

**Review by mkhl — APPROVED** [review]

**Pull request merged**